math/mathcore/inc/Math/IFunction.h

```diff
@@ -216,22 +216,22 @@ namespace ROOT {
          */
          T Derivative(const T *x, unsigned int icoord = 0) const { return DoDerivative(x, icoord); }
 
-      /**
-          Optimized method to evaluate at the same time the function value and derivative at a point x.
-          Often both value and derivatives are needed and it is often more efficient to compute them at the same time.
-          Derived class should implement this method if performances play an important role and if it is faster to
-          evaluate value and derivative at the same time
+         /**
+             Optimized method to evaluate at the same time the function value and derivative at a point x.
+             Often both value and derivatives are needed and it is often more efficient to compute them at the same time.
+             Derived class should implement this method if performances play an important role and if it is faster to
+             evaluate value and derivative at the same time
 
-      */
-      virtual void FdF(const T *x, T &f, T *df) const = 0;
+         */
+         virtual void FdF(const T *x, T &f, T *df) const = 0;
 
-    private:
+      private:
 
 
-      /**
-         function to evaluate the derivative with respect each coordinate. To be implemented by the derived class
-      */
-      virtual T DoDerivative(const T *x, unsigned int icoord) const = 0;
+         /**
+            function to evaluate the derivative with respect each coordinate. To be implemented by the derived class
+         */
+         virtual T DoDerivative(const T *x, unsigned int icoord) const = 0;
```

Comment:
Sorry, these reverts and re-reverts are confusing. I try to go commit-by-commit, but I'm not seeing the final thing, so maybe I'm commenting on stuff that's not actually there.